xml/System.Drawing.Printing/PrinterSettings.xml

```diff
@@ -1654,13 +1654,16 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets a value indicating whether this printer supports color printing.</summary>
+        <summary>On .NET Frameworks 1.1 to 4.8, .NET Core framework versions 3.1 and 3.0, and on .NET Framework Extensions 2.1 to 3.1, gets a value indicating whether this printer uses multiple adjacent color bits for each pixel. This value does not guarantee if this printer supports color.<br/>
+          On .NET 5, gets a value indicating whether this printer supports color printing.</summary>
         <value>
-          <see langword="true" /> if this printer supports color; otherwise, <see langword="false" />.</value>
+          On versions below .NET 5, <see langword="true" /> if this printer uses multiple adjacent color bits for each pixel; otherwise, <see langword="false" />.
+          On .NET 5 and above, <see langword="true" /> if this printer supports color; otherwise, <see langword="false" />.</value>
```

Comment:
```suggestion
          .NET 5 and later versions: <see langword="true" /> if this printer supports color; otherwise, <see langword="false" />.</value>
```